package.json

```diff
@@ -28,7 +28,9 @@
     "package-all": "yarn build && build -mwl",
     "package-all-dev": "yarn build-dev && build -mwl",
     "lint": "tslint --project tsconfig.json --force app/**/*.ts typings/**/*.ts test/**/*.ts",
-    "clean": "rimraf release dist node_modules",
+    "clean": "rimraf release dist; yarn clean-deps",
```

Comment:
@csmoe now that we provide `clean-deps` and `reinstall` I think we could leave `clean` is it was before (without `clean-deps`), if not any of `clean`, `clean-deps` or `reinstall` would force you to reinstall all packages. Don't you think @aesedepece ?

Reply:
That makes sense.

If you do `clean-deps` and then `clean` or viceversa, it doesn't fail because commands are joined with `;`, yet it shows an ugly error because `rimraf` gets removed.

I support @anler's suggestion.